code/modules/projectiles/projectile.dm

```diff
@@ -690,14 +692,22 @@
 		START_PROCESSING(SSprojectiles, src)
 	pixel_move(1, FALSE)	//move it now!
 
-/obj/projectile/proc/setAngle(new_angle)	//wrapper for overrides.
+/obj/projectile/proc/setAngle(new_angle) //wrapper for overrides.
 	Angle = new_angle
 	if(!nondirectional_sprite)
 		var/matrix/M = new
 		M.Turn(Angle)
 		transform = M
 	if(trajectory)
 		trajectory.set_angle(new_angle)
+	if(fired && hitscan && isloc(loc) && (loc != last_angle_set_hitscan_store))
+		last_angle_set_hitscan_store = loc
+		var/datum/point/pcache = new (src)
+		var/list/L = list() 
+		L = trajectory.return_coordinates()
```

Comment:
Better variable name. Also, why are you not just doing `L = trajectory.return_coordinates()`?

Reply:
Fixed, thanks